### ISSUE TITLE
Heatmap Calendar does not render correctly with ITS-Theme [Bug] 

### ISSUE THREAD

**Issue by lth29** [SlRvb/Obsidian--ITS-Theme#102]
### Describe the bug
Heatmap Calendar does not render well. Once you write down the code, you can see many bullet points around and some of them overlap the calendar boxes.

**Expected behavior**
Calendar without bullet points around.

### To Reproduce
1. Install Heatmap Calendar and Dataview
2. Create a folder called Example and a new page called 2022-04-18 with metadata _read::1_.
3. and copy the next code adding _dataviewjs_ at the beginning.

``` dataviewjs
dv.span("**🏋️ Exercise 🏋️**")

const calendarData = {
    year: 2022,
    colors: {
        red: ["#ff9e82","#ff7b55","#ff4d1a","#e73400","#bd2a00"]
    },
    entries: []
}

for(let page of dv.pages('"Example"').where(p=>p.read).sort(p=>p.file.name)){
    calendarData.entries.push({
        date: page.file.name,
        intensity: page.read
    })
       
}

renderHeatmapCalendar(this.container, calendarData)

```

### Screenshots
![Image](https://user-images.githubusercontent.com/88536506/163791215-8d5281db-15c4-466d-a2ff-badc1a3f8a81.png)

### OS
Where does the bug occur?

- [x] Desktop
- [?] Mobile

Obsidian Version:
v0.14.5

**Comment by SlRvb** [comment]
I believe this should be fixed now b3e7ee0

**Comment by lth29** [comment]
yeah, it's working correctly now. Thank you so much.
